docs/documentation/how-tos/manage-xmpp-accounts.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Manage XMPP Accounts
+---
+
+This page described how to manage your XMPP accounts (**not the accounts of your contacts**).
+Therefore, the information are only relevant if you want to replace an old account with a new one
+you want to use multiple accounts.
+
+{% capture eclipse %}
+## Saros Account Preferences
+
+Most of the following configuration steps are located in the
+Saros account preferences. You can open these preferences:
+* With opening `Window > Preferences` and selecting the `Saros` entry. **OR**
+* With opening the "Saros" drop-down menu in the menu bar and clicking on option `Preferences`. **OR**
+* With opening the drop-down menu beside the "Connect" button (![connect icon](../images/icons/connect.png)) in the Saros view
+  and clicking on option `Configure Accounts...`.
+
+## Add an Account
+
+- In the top bar of the Saros view, open the drop-down menu beside the "Connect" button (![connect icon](../images/icons/connect.png)).
+- Select option `Add Account...`.
+- Follow the steps of the wizard.
+
+*Alternatively* via menu bar:
+
+- Open the "Saros" drop-down menu in the menu bar.
+- Select option `Add Account...`.
+- Follow the steps of the wizard.
+
+*Alternatively* via preferences:
+
+- Open the [Saros account preferences](#saros-account-preferences)
+- Click on `Add Account`.
+- Follow the steps of the wizard.
+
+## Remove an Account
+
+- Open the [Saros account preferences](#saros-account-preferences)
+- Select the account you want to remove in the `XMPP Accounts` list.
+- Click on `Remove Account`.
+
+## Modify an Account
+
+- Open the [Saros account preferences](#saros-account-preferences)
+- Select the account you want to modify in the `XMPP Accounts` list.
+- Click on `Edit Account`.
+- Change the JID or the password (if you want to change network specific information open the `Advanced Options`)
+- Click on `Finish`
+
+## Change the Default Account
+
+- Open the [Saros account preferences](#saros-account-preferences)
+- Select the account you want to define as default account.
+- Click on `Activate Account`
+
+{% endcapture eclipse %}
+{% capture intellij %}
+## Add an Account
+
+- In the top bar of the Saros view, click the "Connect" button (![connect icon](images/icons/connect.png)).
+- Choose `Add account...` from the pop-up menu.
+
+{% alert warning %}
+## Note
+As mentioned in the section [missing features](/releases/saros-i_0.2.2.html#missing-features), Saros for IntelliJ **does currently not support the creation, management or deletion of XMPP accounts**. As only people on your friends list can be invited to join your Saros session, you will have to create an account and add friends to your friends list through a different client. Any XMPP client (including Saros for Eclipse) can be used for this purpose.
```

Comment:
Shame on me for forgetting the oxford coma in the original :wink:. 

Also, the part about having to add contacts through other clients is not true anymore. This feature has been added with #785. There also already is a section on how to add an contacts in Saros/I in the old and new getting started page, so it seems like I just forgot to remove this section.

```suggestion
As mentioned in the section [missing features](/releases/saros-i_0.2.2.html#missing-features), Saros for IntelliJ **does currently not support the creation, management, or deletion of XMPP accounts**.
```